SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1441 @@
+		Set-NsxtEdgeNodeAuthenticationPolicy -server $ppmVariables.sddcManagerFqdn -user $ppmVariables.sddcManagerUser -pass $ppmVariables.sddcManagerPass -domain $ppmVariables.sddcDomainName -cliLockoutPeriod $autoUnlockIntervalSec -cliMaxAttempt $numAttempts -minPasswdLength $minCharacterLength | Out-Null
+		Write-LogMessage -type INFO -Message "Setting Password Policy on NSX-T Edge Nodes.  Complete..."
+	}
+	
+	#WSA
+	if ($ppmVariables.wsaFqdn -and $ppmVariables.wsaAdminUser -and $ppmVariables.wsaAdminPass) {
+		if($ppmEnvironmentalDetails | where { ($_.productType -match "WSA") -and ($_.productAttributes.driftAlarm -match "Red") } ) {
+			$products = $ppmStandardConfigValues | Select-Object -Skip 0 | Where-Object -FilterScript { $_.productType -eq "WSA" }
+			$minCharacterLength = $products.productAttributes.passwdMinimumLength
+			$minLowercaseChar = $products.productAttributes.passwdMinLowercase
+			$minUppercaseChar = $products.productAttributes.passwdMinUppercase
+			$minNumericChar = $products.productAttributes.passwdMinNumeric
+			$minSpecialChar = $products.productAttributes.passwdMinSpecial
+			$passwdHistory = $products.productAttributes.passwdHistoryRestriction
+			$maxConsecutiveIdenticalChar = $products.productAttributes.passwdMaxConsecutiveIdenticalChar
+			$maxPreviousPasswordCharactersReused = $products.productAttributes.passwdMaxPreviousCharactersReused
+			$tempPasswordTtlInHrs = $products.productAttributes.tempPasswdLifetimeInHour
+			$passwordTtlInDays = $products.productAttributes.passwdExpInDays
+			$notificationThresholdInDays = $products.productAttributes.passwdExpReminderInDay
+			$notificationIntervalInDays = $products.productAttributes.passwdExpReminderNotificationFrequencyInDay
+			$numAttempts = $products.productAttributes.passwdMaxFailAttempts
+			$attemptIntervalInMins = $products.productAttributes.passwdAttemptsIntervalInMins
+			$unlockIntervalInMins = $products.productAttributes.passwdUnlockIntervalInMins
+			Write-LogMessage -type INFO -Message "Setting Password Policy on standalone WSA.  Starting..."
+			Start-Sleep -s 10
+			#Write-Host "Requesting WSA Authenication Token" -ForegroundColor Yellow
+			Request-WSAToken -fqdn $ppmVariables.wsaFqdn -user $ppmVariables.wsaAdminUser -pass $ppmVariables.wsaAdminPass | Out-Null
+			#Write-Host "set WSA Password Policy" -ForegroundColor Yellow
+			Set-WSAPasswordPolicy -minLen $minCharacterLength -minLower $minLowercaseChar -minUpper $minUppercaseChar -minDigit $minNumericChar -minSpecial $minSpecialChar -history $passwdHistory -maxConsecutiveIdenticalCharacters $maxConsecutiveIdenticalChar -maxPreviousPasswordCharactersReused $maxPreviousPasswordCharactersReused -tempPasswordTtlInHrs $tempPasswordTtlInHrs -passwordTtlInDays $passwordTtlInDays -notificationThresholdInDays $notificationThresholdInDays -notificationIntervalInDays $notificationIntervalInDays | Out-Null
+			#Write-Host "set WSA Lockout policy" -ForegroundColor Yellow
```

Comment:
Remove the Write-Host comments.

Reply:
addressed in 99029d5ec8015e0c5c645e9abc7c3384e1b3091e